eng/Configurations.props

```diff
@@ -0,0 +1,4 @@
+<Project>
+    <!-- Turn off end of life target framework checks as we intentionally build older .NETCoreApp configurations. -->
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
```

Comment:
Let's not add a new file for this. We can just set this in `Directory.Build.props` in the dotnet/runtime repo it is in  `Configurations.props` as that is the shared file across all pieces for this kind of settings.